projects/storefrontlib/src/shared/components/form/form-errors/form-errors.component.ts

```diff
@@ -16,33 +20,27 @@ import { map, startWith } from 'rxjs/operators';
   templateUrl: './form-errors.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class FormErrorsComponent {
-  _control: FormControl;
+export class FormErrorsComponent implements OnChanges {
+  _formGroup: FormGroupDirective;
   errors$: Observable<string[]>;
 
+  constructor(@Optional() @Host() protected formGroup: FormGroupDirective) {
+    this._formGroup = formGroup;
```

Comment:
why? We can just access `this.formgroup` in this controller, no need to store it in a local var, right?